irohad/simulator/impl/simulator.cpp

```diff
@@ -92,21 +94,36 @@ namespace iroha {
           });
     }
 
-    void Simulator::process_verified_proposal(model::Proposal proposal) {
+    void Simulator::process_verified_proposal(
+        const shared_model::interface::Proposal &proposal) {
       log_->info("process verified proposal");
+
       model::Block new_block;
```

Comment:
Block will be replaced with crypto_provider relocation. Now it depends on signature and IR-968 will resolve this dependency.